README.md

```diff
@@ -18,40 +18,23 @@ body) of requests. It also does not aim to be the fastest, or most secure.
 
 Java (version 16 or later). No third-party libraries required for execution.
 
-Dependencies for build / test (such as JUnit) are managed using Gradle. All dependencies should be current, stable
-versions.
+## Running
 
-## Building
-
-### With Gradle
-
-ProxyAuth can be built using Gradle, and includes the
-[Gradle Wrapper](https://docs.gradle.org/7.2/userguide/gradle_wrapper.html). Note that Gradle has different
-requirements, such as not currently supporting Java 17 or later.
-
-- On Windows:
-  `gradlew build`
-- On Linux / Unix:
-  `./gradlew build`
-
-Gradle will create the jar file in `build\libs` (e.g. `build\libs\ProxyAuth-0.1.0.jar`)
-
-### Without Gradle
+TODO - depends on https://github.com/Zeckie/ProxyAuth/issues/6 (configuration)
 
-It does not rely on any third-party libraries, so can be built using any Java IDE, or even javac.
+ProxyAuth does not have a GUI, and should be run from a console / shell such as `bash` / `cmd.exe`.
 
-`C:\example\ProxyAuth\src\main\java>javac -d ..\bin proxyauth\*.java`
+### Windows
 
-## Installation
+`java -jar ProxyAuth-0.1.0.jar`
 
-TODO - depends on https://github.com/Zeckie/ProxyAuth/issues/6 (configuration)
+### Linux/MaxOS
 
-ProxyAuth does not have a GUI, and should be run from a console / shell such as `cmd.exe`.
-`java -jar ProxyAuth-0.1.0.jar`
+`ProxyAuth-0.1.0.jar`
```

Comment:
Do most Linux and MaxOS installations support this out of the box (without having to install extra tools such as `binfmt-support`)?

Reply:
Which OSs / distros have you tested this on?

Reply:
hmm, I don't know where I saw this happening. yeah, this isn't a good one.